src/doc/CMakeLists.txt

```diff
@@ -56,14 +60,21 @@ endif()
 # HTML output directory
 set(SPHINX_HTML_DIR "${CMAKE_CURRENT_BINARY_DIR}/html")
  
-configure_file(
-    "${CMAKE_CURRENT_SOURCE_DIR}/conf.py"
-    "${CMAKE_CURRENT_BINARY_DIR}/conf.py"
-    @ONLY)
- 
+if(WIN32)
+    set(sphinx_launcher ${PYTHON_EXECUTABLE}) 
+endif()
+
 add_custom_target(gui_manual ALL
+    ${sphinx_launcher}
     ${SPHINX_EXECUTABLE}
         -a -q -b html
+        -D html_theme_path=${SPHINX_THEME_DIR}
+        -D html_theme=${SPHINX_THEME}
+        -D version="${VISIT_VERSION_MAJOR}.${VISIT_VERSION_MINOR}"
```

Comment:
version is a specific override for what is in the conf.py file.  I modified this according to what I found in the sphinx-build documenation for overriding conf.py settings.

I did this so that your automatic generation of the docs will work, as well as generation of the docs by a developer in the build environment.

Reply:
Ah, ok, so your intention *is* to override's what ``version`` means in ``conf.py``

Reply:
I didn't think I was overriding what it means, only the value set in conf.py.  If 'version' isn't supposed to be VisIt's version, then it doesn't need to be part of our build.